docs/UserGuide.md

```diff
@@ -118,6 +154,76 @@ Format: `edit_status INDEX s/STATUS`
+
+Examples:
+* `add_docs 1 rs/https://www.example.com/resume cl/https://www.example.com/coverletter` adds the resume link `https://www.example.com/resume` 
+and cover letter link `https://www.example.com/coverletter` to the 1st application in the list of applications.
+* `add_docs 2 rs/https://www.goodresume.com/myresume cl/https://www.goodcoverletter.com/mycoverletter` adds the resume link `https://www.goodresume.com/myresume`
+  and cover letter link `https://www.goodcoverletter.com/mycoverletter` to the 2nd application in the list of applications.
+
+### Edit documents: `edit_docs`
+
+Edits the documents which include the resume link and cover letter link previously added to a specified application.
+
+Format: `edit_docs INDEX rs/RESUME_LINK cl/COVER_LETTER_LINK`
+
+- The index refers to the index number shown in the displayed internship list.
+- The index must be a positive integer 1, 2, 3, …​
+- `RESUME_LINK` must be a valid URL in the format `http://domain/path` or `https://domain/path`.
+- `COVER_LETTER_LINK` must be a valid URL in the format `http://domain/path` or `https://domain/path`.
+
+Examples:
+* `edit_docs 1 rs/https://www.example.com/resume cl/https://www.example.com/coverletter` updates the resume link and cover letter link to `https://www.example.com/resume`
+  and `https://www.example.com/coverletter` respectively for the 1st application in the list of applications.
+* `edit_docs 2 rs/https://www.example.com/resume` updates the resume link to `https://www.example.com/resume` for the 2nd application in the list of applications.
+* `edit_docs 3 cl/https://www.example.com/coverletter` updates the cover letter link to `https://www.example.com/coverletter` for the 3rd application in the list of applications.
+
+### Delete documents: `delete_docs`
+
+Deletes the documents previously added to a specified application.
+
+Format: `delete_docs INDEX`
+
+- The index refers to the index number shown in the displayed internship list.
+- The index must be a positive integer 1, 2, 3, …​
+
+Examples:
+* `delete_docs 1` deletes the documents for the 1st application in the list of applications.
+
+### Archive an internship application: `archive`
+
+Archives a specified application so that it would be hidden from the list of ongoing applications.
+
+Format: `archive INDEX`
+
+- The index refers to the index number shown in the displayed internship list.
+- The index must be a positive integer 1, 2, 3, …​
+
+Examples:
+* `archive 1` archives the 1st application in the list of applications.
+
+### Unarchive an internship application: `unarchive`
+
+Unarchives a specified application that was previously archived so that it would be shown in the list of ongoing applications.
+
+Format: `unarchive INDEX`
+
+- The index refers to the index number shown in the displayed internship list.
+- The index must be a positive integer 1, 2, 3, …​
+
+Examples:
+* `unarchive 1` unarchives the 1st application in the list of applications.
+
```

Comment:
Maybe you have forgotten to write the user guide for the `list_archived` feature? In addition, you can consider to clarify the "internships list" and "list of applications" as "archived internships list" since they are different filtered lists.